src/core_plugins/kibana/public/home/home.less

```diff
@@ -0,0 +1,36 @@
+@import (reference) "~ui/styles/variables.less";
+
+.home {
+  background-color: @globalColorLightestGray;
+  min-height: 100vh;
+}
+
+.homePanel {
+  padding: 24px;
+  background-color: @white;
+}
+
+.topFeatures {
```

Comment:
I'd suggest prefixing your css with the module name. In this case, it seems to be `home`, so everything should be prefixed `home` to avoid collisions.  Or if this is a separate thing, make `feature` its own file.